components/CollectiveCover.js

```diff
@@ -147,10 +151,54 @@ class CollectiveCover extends React.Component {
                 />
                 , &nbsp;
                 <FormattedTime value={props.collective.endsAt} timeZone={props.collective.timezone} />
-                &nbsp; - &nbsp;
+                &nbsp;{' '}
+                {moment()
+                  .tz(props.collective.timezone)
+                  .zoneAbbr()}{' '}
+                (<FormattedMessage id="eventTime" defaultMessage="Event Time" />)
               </React.Fragment>
             )}
-            {get(props.collective, 'location.name')}
+            <br />
+            {this.checkTimeDiff() && (
+              <em>
+                {props.collective.startsAt && (
+                  <React.Fragment>
+                    <FormattedDate
+                      value={props.collective.startsAt}
+                      timeZone={momentTimezone.tz.guess()}
+                      weekday="short"
+                      day="numeric"
+                      month="long"
+                    />
+                    , &nbsp;
+                    <FormattedTime value={props.collective.startsAt} timeZone={momentTimezone.tz.guess()} />
+                    &nbsp; - &nbsp;
+                  </React.Fragment>
+                )}
+
+                {props.collective.endsAt && (
+                  <React.Fragment>
+                    <FormattedDate
+                      value={props.collective.endsAt}
+                      timeZone={momentTimezone.tz.guess()}
+                      weekday="short"
+                      day="numeric"
+                      month="long"
+                      year="numeric"
+                    />
+                    , &nbsp;
+                    <FormattedTime value={props.collective.endsAt} timeZone={momentTimezone.tz.guess()} />
+                    &nbsp;{' '}
+                    {moment()
+                      .tz(momentTimezone.tz.guess())
+                      .zoneAbbr()}{' '}
+                    (<FormattedMessage id="localTime" defaultMessage="Your Time" />)
+                  </React.Fragment>
+                )}
+                <br />
+              </em>
+            )}
+            <FormattedMessage id="location" defaultMessage="Location" /> {get(props.collective, 'location.name')}
```

Comment:
- Let's keep the `:` that you initially added (it looked good)
- Let's also include the event name in the translation string (as per https://docs.opencollective.com/help/contributing/developers/translations#dont-assume-words-order-stays-the-same-in-other-languages, we shouldn't assume that words order stays the same accross languages)
- We shouldn't display the `Location: ` tag if the location is not set

```suggestion
            {get(props.collective, 'location.name') && (
              <FormattedMessage
                id="EventCover.Location"
                defaultMessage="Location: {location}"
                values={{ location: get(props.collective, 'location.name') }}
              />
            )}
```